app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,57 @@
+<p id="notice"><%= notice %></p>
+
+<p>
+  <strong>Name:</strong>
+  <%= @recipe.name %>
+</p>
+
+<p>
+  <strong>Preparation time:</strong>
+  <%= @recipe.preparation_time %>
+</p>
+
+<p>
+  <strong>Cooking time:</strong>
+  <%= @recipe.cooking_time %>
+</p>
+
+<p>
+  <strong>Description:</strong>
+  <%= @recipe.description %>
+</p>
+
+<p>
+  <strong>Public:</strong>
+  <%= @recipe.public %>
+</p>
+
+<p>
+  <strong>User:</strong>
+  <%= @recipe.user_id %>
+</p>
+
+<%= link_to 'Edit', edit_recipe_path(@recipe) %> |
+<%= link_to 'Back', recipes_path %>
+<%= link_to 'Add ingredient', new_recipe_recipe_food_path(@recipe) %>
+<table>
+  <thead>
+    <tr>
+      <th>Food</th>
+      <th>Quantity</th>
+      <th>Value</th>
+      <th>Actions</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @recipe.recipe_foods.each do |recipe_food| %>
+      <tr>
+        <td><%= recipe_food.food.name %></td>
+        <td><%= recipe_food.food.measurement_unit %></td>
+        <td><%= recipe_food.food.price * recipe_food.quantity %></td>
+        <td><%= link_to 'Edit', edit_recipe_food_path(recipe_food) %></td>
+        <td><%= button_to 'Destroy', recipe_food_path(recipe_food), method: :delete, data: { confirm: 'Are you sure?' } %></td>
+      </tr>
+    <% end %>
```

Comment:
- [ ] On this section, you can add a table like this to keep the information

![rails16](https://user-images.githubusercontent.com/55635977/142577775-11f4d0cd-de82-4b00-906c-8317ed2d0bd4.png)

Instead of this

![rails15](https://user-images.githubusercontent.com/55635977/142577130-67aa2b10-0bcd-43aa-952f-bd10d025dc2e.png)

Reply:
Fixed